### ISSUE TITLE
RFE: OFS gridded current models - ability to download

### ISSUE THREAD

**Issue by rgleason** [rgleason/weather_routing_pi#155]
**Is your feature request related to a problem? Please describe.**
Weather_routing can use 30 year averaged ocean currents that are available in the Climatology data and it can use a second RTOFS grib which has gridded data for surface currents available. The problem is that the RTOFS data is not fully implemented for coastal areas. For example, the last I checked, the Gulf of Maine was not available.
At present, weather_routing does not use the point Tide/Current prediction information to interpolate gridded points..
What we need is a predictive gridded model that can be downloaded for near shore weather routing.
OFS appears to be headed in this direction.  See this model.
https://tidesandcurrents.noaa.gov/ofs/ofs_mapplots.html?ofsregion=cb&subdomain=0&model_type=currents_nowcast

Also see page 45 of David Burch's  ENC Essentials booklet and his [Youtube](https://www.youtube.com/watch?v=2t-ED_gLGdc) using luckgrib.  Luckgrib current grib exported to qtVlm and used in qtVlm.  Then compares this data to Noaa tidal current prediction data.

Luckgrib data description of an[ OFS current model for Delaware Bay](https://luckgrib.com/models/ofs_cb/)
Luckgrib data description of an [OFS current model for Gulf of Maine](https://luckgrib.com/models/ofs_gom/)  Also [GoMOFS](https://tidesandcurrents.noaa.gov/ofs/gomofs/gomofs_info.html)

**Describe the solution you'd like**
An OFS gridded current model that can be downloaded and used in weather_routing.
We need the ability to download these grib data models directly.

**Comment by rgleason** [comment]
@sebastien-rosset  Sebastien, it occurs to me that this is a Grib_pi issue and that it should be moved there to OpenCPN grib_pi?  Is that right?

**Comment by sebastien-rosset** [comment]
> @sebastien-rosset  Sebastien, it occurs to me that this is a Grib_pi issue and that it should be moved there to OpenCPN grib_pi?  Is that right?

Yes the first step would be to support this feature in the grib plugin. After it's done potentially there would be some work to do in the WR plugin.or perhaps it would just work depending on how the grib plugin has implemented the feature.

**Comment by rgleason** [comment]
Made a new RFE in opencpn for grib_pi
